views/newsfeed.erb

```diff
@@ -0,0 +1,32 @@
+<head>
+  <title>Chitter</title>
+  <style>
+    body {
+      background-color: lightskyblue;
+      text-align: center;
+    }
+    ul {
+      list-style-position: inside;
+    }
+  </style>
+</head>
```

Comment:
Loving this styling! This looks really cool! I did wonder how to do this without a separate css document but this is a great way!